src/lib.rs

```diff
@@ -182,3 +182,21 @@ fn clockpoly(
     let arcs = [arc_outer, arc_inner].concat();
     Polygon::new(LineString::from(arcs), vec![])
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    #[test]
+    fn internal() {
+        let args: Vec<String> = Vec::new();
+        let da = Params::from_iter(args);
```

Comment:
Why not call this `params`? Matches the name of the type and doesn't take as much guesswork to realize it's "default arguments"

Reply:
:+1:  done.